htdocs/index.php

```diff
@@ -41,6 +41,24 @@
 
 $serverrequest = \Laminas\Diactoros\ServerRequestFactory::fromGlobals();
 
+// Remove the lorispath from the URI query parameters.
+// Both the request query parameters and the URI query string must be updated.
+$params        = $serverrequest->getQueryParams();
+$params        = array_diff_key($params, ["lorispath" => null]);
+$serverrequest = $serverrequest->withQueryParams($params);
+
+$query = implode(
+    "&",
+    array_map(
+        fn ($key, $value) => $key . "=" . $value,
+        array_keys($params),
+        $params
+    )
+);
```

Comment:
I did not use `http_build_query` because the parameters should already be URL-encoded.